components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/webpack.config.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
No need of changing the license year here, since the file was already there before your changes.